src/Microsoft.Diagnostics.Monitoring.WebApi/Utilities.cs

```diff
@@ -62,6 +67,46 @@ public static KeyValueLogScope CreateArtifactScope(string artifactType, IEndpoin
             return scope;
         }
 
+        public static string GetLogsContentType(LogFormat format)
+        {
+            if (format == LogFormat.EventStream)
+            {
+                return ContentTypes.TextEventStream;
+            }
+            else if (format == LogFormat.NDJson)
+            {
+                return ContentTypes.ApplicationNdJson;
+            }
+            else if (format == LogFormat.JsonSequence)
+            {
+                return ContentTypes.ApplicationJsonSequence;
+            }
+            else
+            {
+                return ContentTypes.TextEventStream;
+            }
+        }
+
+        public static async Task StartLogsPipeline(TaskCompletionSource<object> startCompletionSource, LogFormat format, IEndpointInfo endpointInfo, EventLogsPipelineSettings settings, Stream outputStream, CancellationToken token)
```

Comment:
nit: CaptureLogsAsync, to be consistent with the other artifact collection methods.